<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>631</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Практическая работа с камерой и принтером</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="cameraScreen">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Тип товара</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeProduct"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Номер партии</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="partyNumber"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Дата выработки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="date">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Напечатать этикеток</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="countStickers">
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Print">
         <property name="text">
          <string>Начать печать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="Open"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="AboutUs"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Open">
   <property name="text">
    <string>Открыть директорию с DataMatrix</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="AboutUs">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
